app.py

```
@@ -74,15 +80,13 @@ def logout():
 @app.route("/api/mlist")
 def listt():
     if not session:
-        return jsonify({"resulet": "you must loggin", "status": 403})
+        return jsonify({"result": "you must login", "status": 403})
     api_admin = session.get(
         "user"
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)